contributed_definitions/NXem.nxdl.xml

```diff
@@ -62,107 +107,127 @@
-         certain terms they have used are not eventually conceptually similar if not
-         the same as what this application definition and its base classes provide.
+         (EM-research specific base classes) for defining schemas for EM research.
+         Thereby, NXem can be useful to extends top-level ontologies towards a domain-
+         and method-specific ontology for electron microscopy as it is used for
+         materials research.
          
-         We are happy for receiving your feedback.
+         Working towards a common vocabulary is a community activity that profits from
+         everybody reflecting in detail whether certain terms they have used in the past
+         are not eventually conceptually similar if not the same as what this application
+         definition and its base classes provide. We are happy for receiving your feedback.
          
-         It is noteworthy to understand that (not only for) NeXus, schema differ
+         **Addressing the generality versus specificity challenge**:
+         It is noteworthy to understand that (not only for NeXus), schemas differ
          already if at least one field is required in one version of the schema,
-         but it is set optional in another version. If group(s), field(s), or
-         attributes are removed or added, or even a docstring is changed, schemata can
-         become inconsistent. An application definition here serves as a contract
-         between a data provider and a data consumer. These two can be software tools
-         (like the vendor software to drive the instrument or a scientific software
-         for doing artificial intelligence with EM data).
-         Such changes of a schema lead to new versions.
-         
-         Tools like NeXus do not avoid or protect against inconsistencies; however
-         NeXus offers a mechanism and toolset, through which schemata can be
-         documented and defined. In effect, having an openly documented
+         but it is set optional in another schema. If group(s), field(s), or
+         attributes are removed or added, or even a docstring is changed, schemas can
+         become inconsistent. It is noteworthy to mention that the idea of a NeXus application
+         definition serves as a contract between a data provider and a data consumer.
+         Providers can be the software of a specific microscopes or users with specific
+         analysis needs. Consumers can be again specific software tools, like vendor
+         software for controlling the instrument or a scientific software for doing
+         artificial intelligence analyses on EM data). Such changes of a schema lead
+         to new versions. Strictly speaking an application definition can only be
+         fully general if it does not make a single entry required, in which case however
+         it would also not offer much value as even empty datasets would be compliant
+         with such a schema.
```

Comment:
> Strictly speaking an application definition can only be fully general if it does not make a single entry required, in which case however it would also not offer much value as even empty datasets would be compliant with such a schema.

Is this needed?

Reply:
removed

Reply:
Removed also in PR1271